internal/subscriber/subscriber.go

```diff
@@ -87,78 +76,48 @@ func (s *Subscriber) GetID() string {
 
 // GetSubscription ...
 func (s *Subscriber) GetSubscription() string {
```

Comment:
Since `subscription` in subscriber is a subscription model and we are returning the subscription name here, rename this to `GetSubscriptionName` ?